particles/Renderer.cpp

```diff
@@ -259,9 +260,14 @@ void Renderer::init(score::gfx::RenderList& renderer)
                     m_processUBO, 0, sizeof(score::gfx::ProcessUBO), &this->node.standardUBO);
 
         if(!particlesUploaded) {
-            for(int i = 0; i < maxparticles * 4; i++) {
-                data[i] = 50 * double(rand()) / RAND_MAX - 1;
-                speed[i] = (30 * double(rand()) / RAND_MAX - 1) * 0.01;
+            for(int i = 0; i < maxparticles * 4; i+=4) {
+                double angle = 2. * (i/4) * M_PI / (double)n.particlesNumber;
+                data[i] = 5*std::cos(angle);
```

Comment:
all this done in the CPU will take a lot of time, is there not a way to do it on the compute shader instead ?